File: 백준/Bronze/10818. 최소， 최대/최소， 최대.cc
#define _CRT_SECURE_NO_WARNINGS

#include <stdio.h>
#include <iostream>
#include <algorithm>

int min=-1000001;
int max= 1000001;

int main(void)
{
	int count = 0;
	scanf("%d", &count);

	for (int i = 0; i < count; i++)
	{
		int now = 0;
		scanf("%d", &now);

		if (now > max||max== 1000001)
		{
			max = now;
		}

		if (now < min|| min==-1000001)
		{
			min = now;
		}

	}

	printf("%d %d", min, max);
}

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.js
let fs=require('fs');
let input=fs.readFileSync('/dev/stdin').toString().split('\n');

let count=Number(input[0]);
let arr=input[1].split(' ').map(Number);

let max=-1000000;
let min=1000000;

arr.map((value)=>{
    if(value<min){
        min=value;
    }

    if(value>max){
        max=value;
    }
})

console.log(min, max);